### DISCUSSION TITLE
send rejected mails to special mail address

### DISCUSSION THREAD

**Discussion by bikedevone** [rspamd/rspamd#3717] [General]
Hi guys,

first at all your system rocks. :)

I have one question. I want to send rejected mails to a special mail address for further analysis. How can i accomplish that task? Or is there maybe another way to look into the rejected mails?

Thanks in advance.

**Comment by moisseev** [comment]
Use [metadata exporter](https://rspamd.com/doc/modules/metadata_exporter.html#settings-send_mail-backend) module.

Please do not open issues for questions.
